exhibitors/urls.py

```diff
@@ -0,0 +1,18 @@
+from django.urls import re_path as urls
+
+from .views import SettingsView, ExhibitorListView, ExhibitorCreateView, ExhibitorEditView, ExhibitorDeleteView, ExhibitorCopyKeyView
+
+urlpatterns = [
+    urls(r'^control/event/(?P<organizer>[^/]+)/(?P<event>[^/]+)/settings/exhibitors',
```

Comment:
Please use new style of URL conf https://docs.djangoproject.com/en/5.1/ref/urls/#path

Reply:
This was the format of urls.py for other plugins in eventyay-tickets hence followed the same. Should I change to the new style ?